spearhead/assets/sass/_posts-and-pages.scss

```diff
@@ -25,7 +25,10 @@
 
 .site-main > * {
 	margin: 0 auto;
-	padding: calc(3 * var(--global--spacing-vertical)) 0;
```

Comment:
@scruffian do you know why this was set to such a large value and on such a generic selector?

Reply:
This is exactly what I was comenting on slack today. Even buttons are using calculations with this variable as a base (and they look huge because of it). This is inherited from Seedlet, and on Seedlet it's value is 30px.

Reply:
Yeah we do something similar in [Seedlet](https://github.com/Automattic/themes/blob/master/seedlet/assets/sass/structure/_vertical-margins.scss#L45), so this is following that pattern. I agree its problematic.